cpp/tests/io/arrow_io_source_test.cpp

```diff
@@ -25,7 +25,9 @@
 #include <cudf/io/json.hpp>
 #include <cudf/io/parquet.hpp>
 
+#pragma nv_diag_suppress 611
 #include <arrow/filesystem/filesystem.h>
+#pragma nv_diag_default 611
```

Comment:
We won't need this because this file is a `.cpp` file.

Reply:
Good call.